components/datatypes/sessiondetails.schema.json

```diff
@@ -338,11 +333,15 @@
         "xdm:playerName",
         "xdm:channel"
       ]
+    },
+    "@sessionDetailsReporting": {
+      "properties": {
+        "xdm:pccr": {
+          "title": "Pccr",
+          "type": "boolean",
+          "description": "Indicates that a redirect occurred."
+        }
+      }
     }
-  },
-  "allOf": [
-    {
-      "$ref": "#/definitions/sessionDetails"
```

Comment:
Need to add both the local definitions here

Reply:
Done